openshift/scripts/oc_deploy.sh

```diff
@@ -21,13 +21,20 @@ source "$(dirname ${0})/common/common"
 #%   ${THIS_FILE} pr-0 apply
 #%
 
+# Use keycloak prod url when deploying to prod
```

Comment:
@Nkuruba @Kyubinhan again - we don't need this here at all - we don't need to pass it to the template, the template can grab the values from the configmap.

Reply:
@Sybrand now the template is grabbing values from global configmap.